graphing/html/index.html

```diff
@@ -3,60 +3,135 @@
 
 <head>
     <script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.3/dist/Chart.min.js" integrity="sha256-R4pqcOYV8lt7snxMQO/HSbVCFRPMdrhAFMH+vr9giYI=" crossorigin="anonymous"></script>
+    <script src="https://cdn.rawgit.com/chartjs/chartjs-plugin-annotation/master/chartjs-plugin-annotation.min.js"></script>
 </head>
 
 <body>
     <canvas id="graph"></canvas>
     <script>
         fetch('/data').then(d => d.json()).then(data => {
             // From inside of here data will have the data loaded from the server
-            console.log(data)
+            console.log(data);
 
 
-            /**
-             * I just copy pasted ChartJS example from there website.
-             * From here you will just be concerned with rendering the chart to be similar to the one we already have
-             **/
-
             const ctx = document.getElementById('graph').getContext('2d');
+            // Chart.pluginService.register({
+            //     afterDraw: function(chart) {
+            //         if (typeof chart.config.options.lineAt != 'undefined') {
+            //             var lineAt = chart.config.options.lineAt;
+            //             var ctxPlugin = chart.chart.ctx;
+            //             var xAxe = chart.scales[chart.config.options.scales.xAxes[0].id];
+            //             var yAxe = chart.scales[chart.config.options.scales.yAxes[0].id];
+            //
+            //
+            //             if(yAxe.min != 0) return;
+            //
+            //             ctxPlugin.strokeStyle = "grey";
+            //             ctxPlugin.beginPath();
+            //             lineAt = (lineAt - yAxe.min) * (100 / yAxe.max);
+            //             lineAt = (100 - lineAt) / 100 * (yAxe.height) + yAxe.top;
+            //             ctxPlugin.moveTo(xAxe.left, lineAt);
+            //             ctxPlugin.lineTo(xAxe.right, lineAt);
+            //             ctxPlugin.stroke();
+            //         }
+            //     }
+            // });
             const myChart = new Chart(ctx, {
-                type: 'bar',
+                type: 'line',
                 data: {
-                    labels: ['Red', 'Blue', 'Yellow', 'Green', 'Purple', 'Orange'],
+                    labels: ['1', '2', '3', '4', '5', '6', '7','8', '9', '10', '11','12'],
                     datasets: [{
-                        label: '# of Votes',
-                        data: [12, 19, 3, 5, 2, 3],
+                        label: false,
+                        data: [0.6, 0.75, 0.4, 0.65, 0.9, 1.1, 0.78, 1.5, 1.05, 0.4, 0.8, 0.6],
```

Comment:
This data is dynamic so it should be loaded from the API in this file https://github.com/nperez0111/agile-reporting/blob/srokkala-patch-1/graphing/index.js#L4

Right now that is the data variable that you have in the `fetch` call